feature_server/run.py

```diff
@@ -57,6 +57,10 @@ def choose_path(base,top):
 
 # add it to the path so we can import scripts
 sys.path.append(config_dir)
+# add our package to path too so scripts can import `feauture_server/`
```

Comment:
It is true that this importing is a better way and probably something we would advice users/developers to move towards. But for legacy `import commands` should work too.

I'll try to scope the importing to userscripts only (was my intention from the start, but looking at the current implementation it's not here). In userscripts, it is expected that `import map` will import map.py from pyspades and not some https://pypi.python.org/pypi/map